apps/bfd-model/bfd-model-rif-samples/dev/synthea_releases/01-20-2022/characteristics.csv

```diff
@@ -0,0 +1,10001 @@
+Beneficiary Id,MBI Unhashed,Part D Contract Number,Carrier Claims Total,DME Claims Total,HHA Claims Total,Hospice Claims Total,Inpatient Claims Total,Outpatient Claims Total,SNF Claims Total,PartD Events Total
+"-10000000015910,1S00E00DY10,""Z1001,Z1001"",9,0,0,0,0,11,0,2"
```

Comment:
something is wrong with these lines. there are quotes around the entire line and there are two sets of double quotes around the contract list instead just one set.